hbase-protocol-shaded/src/main/protobuf/server/replication/ReplicationServer.proto

```diff
@@ -24,9 +24,21 @@ option java_generic_services = true;
 option java_generate_equals_and_hash = true;
```

Comment:
Does the replication server belong in the core? Should it be done outside in hbase-operator-tools? Just add the communication bits to core?

Reply:
Yes, in the core. The HReplicationServer will be a new role in one HBase cluster.